aos-rpi.yaml

```diff
@@ -364,7 +362,7 @@ parameters:
               conf:
                 - [
                     AOS_HOSTS:append,
-                    " %{AOS_MAIN_NODE_IP}=%{AOS_MAIN_NODE_HOSTNAME}",
+                    " %{AOS_MAIN_NODE_IP}=${AOS_MAIN_NODE_HOSTNAME}",
```

Comment:
Didn't understand how AOS_MAIN_NODE_HOSTNAME can be used after it removal in variables

Reply:
AOS_MAIN_NODE_HOSTNAME has default value in meta-aos config.
Changing %{AOS_MAIN_NODE_HOSTNAME} -> ${AOS_MAIN_NODE_HOSTNAME} means substitution will be done by yocto instead of mouling. So, default value of AOS_MAIN_NODE_HOSTNAME will be used and substituted by yocto during build.